0008-interledger-quoting-protocol/0008-interledger-quoting-protocol.md

```diff
@@ -33,115 +31,68 @@ The sending and receiving ledger are different ledgers. Otherwise, there should
-| `data`   | Quote or Error | An Object describing the quote (when method=`quote_response`) or the error (when method=`error`). |
-
-##### Quote (method = `"quote_response"`)
-
-| Field                         | Type        | Description |
-|:------------------------------|:------------|:------------|
-| `source_connector_account`    | ILP Address | The address of the connector's account in the source ledger where it should receive the first transfer. |
-| `source_ledger`               | ILP Address | The address of the ILP-enabled ledger where the source account is. |
-| `source_amount`               | String      | Integer number amount of currency the connector's account should receive in the source ledger. |
-| `source_expiry_duration`      | String      | Integer number of milliseconds between when the payment in the source ledger is prepared and when it must be executed. |
-| `destination_ledger`          | ILP Address | The address of the ILP-enabled ledger where the destination account is. Equivalent to the
-[`appliesToPrefix`](https://github.com/interledger/rfcs/blob/9664732/asn1/InterledgerQuotingProtocol.asn#L36-L42) field which will possibly be used in the future. |
-| `destination_amount`          | String      | Integer number amount of currency that should be received by the destination account in the destination ledger. |
-| `destination_expiry_duration` | String      | Integer number of milliseconds between when the payment in the destination ledger is prepared and when it must be executed. |
-| `liquidity_curve`             | Number[][]  | Piece-wise linear function that was used to calculate this response, and that applies to the entire destination ledger. |
-
-##### Example response (method = `"quote_response"`)
-
-```js
-{
-  "method": "quote_response",
-  "id": "721e4126-98a1-4974-b35a-8a8f4655f934",
-  "data": {
-    "source_connector_account":"example.eur-ledger.mark",
-    "source_ledger":"example.eur-ledger.",
-    "source_amount":"10025000000",
-    "source_expiry_duration":"6000",
-    "destination_ledger":"example.usd-ledger.",
-    "destination_amount":"9000000000",
-    "destination_expiry_duration":"5",
-    "liquidity_curve":[
-      ["10", "0"],
-      ["100000000000000000", "99799999999999"]
-    ]}
-  }
-}
-```
-
-##### Error (method = `"error"`)
-
-| Field     | Type   | Description |
-|:----------|:-------|:------------|
-| `id`      | String | The type of error. |
-| `message` | String | Additional details about the nature of the error. |
-
-##### Example response (method = `"error"`)
-
-```js
-{
-  "method": "error",
-  "id": "721e4126-98a1-4974-b35a-8a8f4655f934",
-  "data": {
-    "id": "InvalidBodyError",
-    "message": "Missing required parameter: source_address"
-  }
-}
-```
+Quotes are requested with `plugin.sendRequest(message)`, which returns a `Promise` to the quote response (see the Ledger Plugin Interface for more details).
```

Comment:
I'm not sure this should be included in this spec, since how quotes are requested is specific to the ledger plugin interface in the implementation you're using. Maybe just put it in a more generic way that quotes are sent through a request/response mechanism exposed by the ledger plugins or ledger layer.